index.js

```diff
@@ -250,6 +250,48 @@ SteamCommunity.prototype.oAuthLogin = function(steamguard, token, callback) {
 	}, "steamcommunity");
 };
 
+/**
+ * Sign in through Steam on a third party website via OpenID.
+ * @param {string} url - The Steam OpenID login URL to the third party website.
+ * @param {function} callback
+ */
+SteamCommunity.prototype.openidLogin = function(url, callback) {
+	var self = this;
+	self.httpRequestGet({
+		"uri": url,
+		"followAllRedirects": true
+	}, function(err, response, body) {
+		if(err) {
+			callback(err);
+			return;
+		}
+
+		var $ = Cheerio.load(body);
+		var form = $('#openidForm');
+		if (!form) { // Should exist, even if not logged in
+			callback(new Error('Malformed response'));
+			return;
+		}
+
+		if ($('.OpenID_loggedInText').length === 0) {
+			callback(new Error('Not logged in'));
```

Comment:
Everywhere else an error is returned with this message, it's capitalized like `Not Logged In`. Would like to see that here as well for consistency.

Reply:
Like the germans